### ISSUE TITLE
--config issue

### ISSUE THREAD

**Issue by kiliangr** [javascript-obfuscator/javascript-obfuscator#560]
I can't make javascript-obfuscator work with the CLI --config option.

## Current Behavior

throw new ReferenceError('Given config path must be a valid `.js` or `.json` file path');
## Steps to Reproduce

javascript-obfuscator test.js --config conf.js

I've tried to use both json and js files, none of them worked. The only time i go without errors with this command line is with an empty config file.

**Comment by sanex3339** [comment]
show me the content of your config file

**Comment by kiliangr** [comment]
Here is my config file. I used the preset for high obfuscation.


```javascript
module.exports = {
    compact: true,
    controlFlowFlattening: true,
    controlFlowFlatteningThreshold: 1,
    deadCodeInjection: true,
    deadCodeInjectionThreshold: 1,
    debugProtection: true,
    debugProtectionInterval: true,
    disableConsoleOutput: true,
    identifierNamesGenerator: 'hexadecimal',
    log: false,
    renameGlobals: false,
    rotateStringArray: true,
    selfDefending: true,
    shuffleStringArray: true,
    splitStrings: true,
    splitStringsChunkLength: '5',
    stringArray: true,
    stringArrayEncoding: 'rc4',
    stringArrayThreshold: 1,
    transformObjectKeys: true,
    unicodeEscapeSequence: false
};
```

**Comment by sanex3339** [comment]
I cannot reproduce your error. With your config i got
```
ReferenceError: Validation failed. errors:
`splitStringsChunkLength` errors:
    - splitStringsChunkLength must not be less than 1
,    - splitStringsChunkLength must be a number conforming to the specified constraints
```

**Comment by kiliangr** [comment]
Testing it back today, I ended up with the same error you get. After changing `splitStringsChunkLength: '5',` for `splitStringsChunkLength: 5,` everything worked as expected.

**Comment by paulcdejean** [comment]
--config doesn't seem to work on windows...

**Comment by Pixelsplasher** [comment]
Solved by yours truly at #1234

**Comment by Pixelsplasher** [comment]
> --config doesn't seem to work on windows...
Solved by yours truly at #1234
